packages/mc-scripts/src/commands/config:sync.js

```diff
@@ -0,0 +1,138 @@
+const omit = require('lodash/omit');
+const prompts = require('prompts');
+const chalk = require('react-dev-utils/chalk');
```

Comment:
Here we are relying in the fact that `react-dev-utils` dependency currently export `chalk`.
Given that we are going to use [that dependency](https://www.npmjs.com/package/chalk) directly I guess it would be safer to explicitly depend on it by itself declaring it in our `package.json`.